pr_agent/algo/pr_processing.py

```diff
@@ -78,9 +78,9 @@ def get_pr_diff(git_provider: GitProvider, token_handler: TokenHandler, model: s
     return final_diff
 
 
-def pr_generate_extended_diff(pr_languages: list, token_handler: TokenHandler,
-                              add_line_numbers_to_hunks: bool) -> \
-        Tuple[list, int]:
+def pr_generate_extended_diff(pr_languages: list,
+                              token_handler: TokenHandler,
+                              add_line_numbers_to_hunks: bool) -> Tuple[list, int, list]:
```

Comment:
**Suggestion:** Consider adding a docstring to the `pr_generate_extended_diff` function to explain its purpose, arguments, and return value.
```suggestion
def pr_generate_extended_diff(pr_languages: list,
                              token_handler: TokenHandler,
                              add_line_numbers_to_hunks: bool) -> Tuple[list, int, list]:
    """
    Generates an extended diff for the PR languages.

    Args:
        pr_languages (list): The list of PR languages.
        token_handler (TokenHandler): The token handler.
        add_line_numbers_to_hunks (bool): Whether to add line numbers to hunks.

    Returns:
        Tuple[list, int, list]: The extended patches, total tokens, and extended patches tokens.
    """
```